vxyowsup/tests/test_whatsapp.py

```diff
@@ -85,17 +90,35 @@ def assert_ack(self, ack, message, whatsapp_id):
         self.assertEqual(ack.payload['user_message_id'], message['message_id'])
         self.assertEqual(ack.payload['sent_message_id'], whatsapp_id)
 
+    def assert_receipt(self, receipt, message, whatsapp_id):
+        # event_type is 'ack' for some reason
+        self.assertEqual(receipt.payload['event_type'], 'delivery_report')
+        self.assertEqual(receipt.payload['sent_message_id'], whatsapp_id)
```

Comment:
`delivery_report` events don't have a `sent_message_id` -- just a `user_message_id` and a `delivery_status`.